### PR TITLE
v2: commands message sign and verify updated

### PR DESCRIPTION
**Pull request by CryptologySAC**
This PR updates the message sign and verify commands to use the V2 ARK - Crypto package instead of arkjs.

### CODE REVIEW

**Comment by faustbrian** [comment]
Please use the same fixture as all crypto packages use for message signing.

https://github.com/ArkEcosystem/php-crypto/blob/master/tests/fixtures/message.json

File: lib/utils/crypto.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+'use strict'
+const arkecosystem = require('@arkecosystem/crypto')
+const arkUtils = arkecosystem.utils
+const ECPair = arkecosystem.ECPair
+const ECSignature = arkecosystem.ECSignature
+const crypto = arkecosystem.crypto
+
+class Crypto {
+  sign (msg, passphrase, networkVersion) {
```
Comment (faustbrian) [inline lib/utils/crypto.js]: Please rename `msg` to `message`, no gains by having those abbreviation.

**Comment by faustbrian** [comment]
@wownmedia Please revert the last commit as it should be part of a new PR.

**Comment by CryptologySAC** [comment]
Reverted, but next PR does include commits from this one as they share some
changes. I'll hold off on submitting it until after this one is merged, so
it doesn't get too confusing.

**Comment by faustbrian** [comment]
You should work on fresh branches for each different group of changes to avoid issues like that, will make your life a lot easier.

**Comment by CryptologySAC** [comment]
I know, but since the address and the sign/verify are basically the same
tech I combined them :)

**Pull request merged**